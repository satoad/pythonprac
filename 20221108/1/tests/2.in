a = DivStr("XcsajdjijaoisjijoDfQWEasdERTdfgRTY")
print(* a // 10)
print(a % 10)