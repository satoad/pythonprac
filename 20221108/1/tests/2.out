Xcs ajd jij aoi sji joD fQW Eas dER Tdf
gRTY
